Celeste.Mod.mm/Patches/Autotiler.cs

```diff
@@ -466,3 +489,22 @@ private class patch_Masked {
 
     }
 }
+
+namespace MonoMod {
+    [MonoModCustomMethodAttribute(nameof(MonoModRules.PatchAutotilerCtor))]
+    class PatchAutotilerCtorAttribute : Attribute { }
+
+    static partial class MonoModRules {
+        public static void PatchAutotilerCtor(ILContext context, CustomAttribute attrib) {
```

Comment:
it'd be nice to have a comment showing what this patch does (preferably a diff of the il like on some of my patches), especially since this deals with locals by number